modules/secrets.js

```diff
@@ -36,8 +37,34 @@ function getSecret(name) {
     }
 }
 
+// @TODO: Make all process.env vars consistent case
```

Comment:
Think this is down to Travis, could change them all there when this goes out?

Reply:
Good point, will merge this as is and then double check / clean things up soon.